lib/Cleantalk/ApbctWP/WcSpamOrdersFunctions.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Cleantalk\ApbctWP;
+
+use Cleantalk\ApbctWP\Variables\Post;
+
+class WcSpamOrdersFunctions
+{
+    public static function restoreOrderAction()
+    {
+        check_ajax_referer('ct_secret_nonce');
```

Comment:
Use `apbct__check_admin_ajax_request` instead of `check_ajax_referer`

Reply:
Done